lib/remotipart/render_overrides.rb

```diff
@@ -13,8 +13,7 @@ def self.included(base)
     def render_with_remotipart *args
       render_without_remotipart *args
       if remotipart_submitted?
-        textarea_body = response.content_type == 'text/html' ? html_escape(response.body) : response.body
-        response.body = %{<textarea data-type=\"#{response.content_type}\" data-status=\"#{response.response_code}\" data-statusText=\"#{response.message}\">#{textarea_body}</textarea>}
+        response.body = %{<script type=\"text/javascript\">try{}catch(err){}</script>#{response.body}}
```

Comment:
@bsimpson We don't need the try..catch here i just left it as is from the remotipart master. If at all we need it we can use it.